orc8r/cloud/deploy/terraform/orc8r-helm-aws/templates/orc8r-values.tpl

```diff
@@ -184,7 +184,6 @@ nms:
       tag: "${docker_tag}"
 
     env:
-      api_host: ${api_hostname}
```

Comment:
In this case, I wanted to understand the implications of using service name directly instead of aws or other LB here. Can you help me understand if there will be a change in how the requests will be load balanced in this case.

Reply:
Thanks for the comment on not bumping chart versions. I can take care of that.

Regarding how loadbalancing works, it's identical to how loadbalancing works externally. In both cases, kube-proxy is what actually gets hit by in-cluster DNS requests and loadbalancers. The loadbalancing strategy will be the same (such as session persistence or round robin mode) for in-cluster requests as with external LB requests. All that changes here are two things:
the DNS name
traffic does not need to leave the k8s cluster, go to the loadbalancer, then get routed back in